flink/src/test/java/io/delta/flink/sink/DeltaSinkStreamingExecutionITCase.java

```diff
@@ -122,9 +122,10 @@ public void runDeltaSinkTest() throws Exception {
         // GIVEN
         DeltaLog deltaLog = DeltaLog.forTable(DeltaSinkTestUtils.getHadoopConf(), deltaTablePath);
         List<AddFile> initialDeltaFiles = deltaLog.snapshot().getAllFiles();
+
         long initialVersion = deltaLog.snapshot().getVersion();
         int initialTableRecordsCount = TestParquetReader.readAndValidateAllTableRecords(deltaLog);
-        assertEquals(initialDeltaFiles.size(), 2);
+        assertEquals(2, initialTableRecordsCount);
```

Comment:
Messed expected with actual + now I'm reading number of records. 
New table contains only one parquet file but still two records.